src/doc/DocDrawingThread.js

```diff
@@ -31,6 +34,15 @@ class DocDrawingThread extends DrawingThread {
         this.reconstructBrowserCoordFromLocation = this.reconstructBrowserCoordFromLocation.bind(this);
     }
 
+    /**
+     * Resets current buffering state.
+     *
+     * @return {void}
+     */
+    resetBufferingState = () => {
```

Comment:
If you make this more generic `setBufferingState(true/false)` you can reuse it on line 70. You could also make it toggle instead.

Reply:
@jeremypress  I thought about that, but I don't want to create the overhead of creating new functions every mouse move event, to be used by `requestAnimationFrame()`